Makefile

```diff
@@ -23,6 +23,7 @@ format:
 install:
 	@poetry install
 	@pre-commit install
+	@pre-commit install-hooks
```

Comment:
@hebertjulio I found the problem! We also need to run `pre-commit install --hook-type commit-msg` to install it properly

https://jorisroovers.com/gitlint/#using-gitlint-through-pre-commit

![image](https://user-images.githubusercontent.com/2728804/128770921-5ccbe4e9-1ee3-4090-93f5-dd442256e55e.png)

Now it works:

```shell
▶ gcmsg 'Test'
black................................................(no files to check)Skipped
isort................................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
black................................................(no files to check)Skipped
isort................................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
gitlint..................................................................Failed
- hook id: gitlint
- exit code: 3

1: CT1 Title does not start with one of fix, feat, chore, docs, style, refactor, perf, test, revert, ci, build: "Test"
1: CT1 Title does not follow ConventionalCommits.org format 'type(optional-scope): description': "Test"
1: T8 Title is too short (4<5): "Test"
```

```suggestion
	@pre-commit install-hooks
	@pre-commit install --hook-type commit-msg
```

Reply:
Yes! I forget to add the command to install commit-msg hook. Thanks! :smile: